dev/sg/cloudsqlproxy/binary.go

```diff
@@ -0,0 +1,90 @@
+// Initially copy-pasta from https://github.com/sourcegraph/controller/blob/main/internal/cloudsqlproxy/gen.go
+package cloudsqlproxy
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/std"
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+	"github.com/sourcegraph/sourcegraph/lib/output"
+)
+
+const CloudSQLProxyVersion = "2.1.1"
```

Comment:
might as well upgrad to the latest

```suggestion
// CloudSQLProxyVersion can be found from https://github.com/GoogleCloudPlatform/cloud-sql-proxy/releases
const CloudSQLProxyVersion = "2.8.1"
```